### PR TITLE
e2e - Automated test for form app

### PR DESCRIPTION
**Pull request by howard-zhang**
Added automated test for form app

### CODE REVIEW

File: apps/form-app-e2e/.eslintrc.json [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+        "@typescript-eslint/camelcase": "off",
+        "no-undef": "off"
+      }
     }
+  ],
+  "ignorePatterns": [
+    "!**/*"
   ]
-}
+}
```
Comment (LanaDIO) [inline apps/form-app-e2e/.eslintrc.json:29]: remove space at the end of the file

File: apps/form-app-e2e/src/integration/forms.feature [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    And the user views the summary of "Personal Information" with "1970-10-30" as "not required" "Birthday"
+    And the user views the summary of "Additional Information" with "No" as "required" "Are you married?"
+    And the user views the summary of "Additional Information" with "Yes" as "not required" "Citizen"
+    And the user views the summary of "Additional Information" with "John" as "required" "First name:Dependant"
+    And the user views the summary of "Additional Information" with "Smith" as "required" "Last name:Dependant"
+    And the user views the summary of "Additional Information" with "2010-01-15" as "not required" "Dob:Dependant"
+    When the user clicks submit button in the form
+    Then the user views a callout with a message of "We're processing your application"
+    When the user sends a delete form request
+    Then the new form is deleted
```
Comment (LanaDIO) [inline apps/form-app-e2e/src/integration/forms.feature:40]: remove space at the end of the file

File: apps/form-app-e2e/tsconfig.e2e.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "cypress",
+      "node",
+      "cypress-axe"
+    ]
+  },
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.js"
+  ]
+}
```
Comment (LanaDIO) [inline apps/form-app-e2e/tsconfig.e2e.json:17]: remove space at the end of the file

File: apps/form-app-e2e/tsconfig.json [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true
   },
-  "include": ["src/**/*.ts", "src/**/*.js"]
-}
+  "include": [
+    "**/*.ts"
+  ]
+}
```
Comment (LanaDIO) [inline apps/form-app-e2e/tsconfig.json:22]: remove space at the end of the file

**Review by LanaDIO — COMMENTED** [review]
remove space at the end of the files where is indicated

File: apps/form-app-e2e/.eslintrc.json (hunk 1 above)
Comment (howard-zhang) [reply apps/form-app-e2e/.eslintrc.json:29]: This file doesn't have empty space or empty line in the end.

File: apps/form-app-e2e/src/integration/forms.feature (hunk 2 above)
Comment (howard-zhang) [reply apps/form-app-e2e/src/integration/forms.feature:40]: This file doesn't have empty space or empty line in the end.

File: apps/form-app-e2e/tsconfig.e2e.json (hunk 3 above)
Comment (howard-zhang) [reply apps/form-app-e2e/tsconfig.e2e.json:17]: This file doesn't have empty space or empty line in the end.

File: apps/form-app-e2e/tsconfig.json (hunk 4 above)
Comment (howard-zhang) [reply apps/form-app-e2e/tsconfig.json:22]: This file doesn't have empty space or empty line in the end.

**Pull request merged**